### DISCUSSION TITLE
How to get RTCP Sender Report from Mediamtx?

### DISCUSSION THREAD

**Discussion by obescha** [bluenviron/mediamtx#1951] [Questions]
Hello everyone.

I'm trying to find the wall clock timestamp from an RTP/RTSP stream thru Mediamtx. In order to do so, Mediamtx should generate a Sender Report RTCP packet, or at least pass one thru from the source stream. Currently I see no such packets in the stream in wireshark. Is there a way to achieve this?

Thank you.

**Comment by aler9** [comment]
Hello, there's an open issue here: #1300
